package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "auth0-authz-extension",
-  "version": "2.8.1",
+  "version": "2.8.2",
```

Comment:
Please remove this, as I will bump the version in another PR. (Also we need to bump in webtask.json)

Reply:
Done